### ISSUE TITLE
Bug: Batch messages with datetime present fail during bulk insert.

### ISSUE THREAD

**Issue by BuzzCutNorman** [BuzzCutNorman/target-mssql#17]
#### Initial Description
When batch messages for a stream contain a datetime field they fail to be loaded by the `bulk_insert_records()` function with the following error message.
```
('22007', '[22007] [Microsoft][ODBC Driver 18 for SQL Server][SQL Server]Conversion failed when converting date and/or time from character string. (241) (SQLExecDirectW)')
```
The same exact stream can be loaded without issue when the batch configuration is removed.  When you check the records just before they are sent to the `insert` portion of `bulk_insert_records()` you can see that non-batch message is a `datetime.datetime` type while the batch message record is a string type.  

**Non-Batch**
`{'Id': 1, 'TestColumn3': datetime.datetime(2018, 11, 13, 20, 20, tzinfo=tzutc())} `
**Batch** 
`{'Id': 1, 'TestColumn3': '2018-11-13T20:20:00+00:00'}`

**Comment by BuzzCutNorman** [comment]
It has taken quite a while to find where the `datetime` was being converted during a normal or non-batch load.  The `Sink` class has a method `_parse_timestamps_in_record` that runs before a the `preprocess_record` method.  When batch mode is used it skips the `_parse_timestamps_in_record` method so a datetime string in iso format is given directly to SQL server which it rejects.

**Comment by BuzzCutNorman** [comment]
This was resolved in version 0.1.0 .
